dll/win32/profmap/profmap_main.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * PROJECT:         ReactOS system libraries
+ * LICENSE:         GPL - See COPYING in the top level directory
```

Comment:
Please use the new style file header format!

Reply:
Fixed. Thanks.
Sorry, not immediately noticed. :sweat_smile: